eng/ApiListing.targets

```diff
@@ -0,0 +1,53 @@
+<Project>
+
+  <Target Name="GenerateApiListing">
```

Comment:
Why generate and check the api ref? Is it just a caching mechanism? We could also pull down the last package and do a diff and api check. I was thinking that is likely what we would do when we enable the api breaking change checking.

Reply:
This is something I knew how to do :) 

Yes, I check them in as package mechanism as pulling the previous version of package requires defining what "previous" is and keeping that up-to-date etc.

I'm happy to switch to that when we enable breaking change checking but wanted something quick because new changes to GA libraries are coming in

Reply:
Previous version should be the latest stable version. This should be fine for now.

Reply:
Yes, but there are two goals:
1. Being aware of what API changes are happening - they are often hard to discern from large code change
2. Making sure they are not breaking